packages/skeleton-react/package.json

```diff
@@ -40,6 +40,7 @@
 		"!./dist/schemas.d.ts"
 	],
 	"dependencies": {
+		"@zag-js/switch": "^0.62.1",
```

Comment:
This should be moved to devDependencies as well

Reply:
Svelte sadly doesn't have a random ID generator built in, so we'll have to implement our own.

Reply:
A local `useId` util function has been implemented into the `skeleton-svelte` package.